doc/release-notes.md

```diff
@@ -141,7 +141,8 @@ Updated settings
 
 - Netmasks that contain 1-bits after 0-bits (the 1-bits are not contiguous on
   the left side, e.g. 255.0.255.255) are no longer accepted. They are invalid
-  according to RFC 4632.
+  according to RFC 4632. Netmasks are used in the `-rpcallowip` parameter and
```

Comment:
- perhaps s/parameter/option|configuration option/ (`Usage:  bitcoind [options]`)

- per `bitcoind -help-debug | grep -C3 -i cidr` and `git grep -ni netmask`, maybe verify if netmasks are also used for the `-whitelist` option

Reply:
Updated, thanks!